pallets/gear/src/weights.rs

```diff
@@ -1,6 +1,6 @@
 // This file is part of Gear.
```

Comment:
seems better to add all */weights.rs files in .gitattributes to proper repo heart-beat